images/Dockerfile.recovery-controller

```diff
@@ -0,0 +1,11 @@
+FROM golang:alpine AS build-env
```

Comment:
Let's not use `latest` or `alpine` but at least a major release of Go. E.g. `golang:1.14-alpine`.